### ISSUE TITLE
🔗 🐞 Copy & pasting with keyboard shortcuts breaks accordion widget (Duplicate of #25861) - [ED-14553]

### ISSUE THREAD

**Issue by vgstef** [elementor/elementor#27030]
### Prerequisites

- [X] I have searched for similar issues in open and closed tickets and cannot find a duplicate.
- [X] I have troubleshooted my issue, and it still exists against the latest stable version of Elementor.

### Description

If we copy and paste an accordion widget using copy and paste (with Ctrl+V), the widget is dysfunctional.

Once pasted, if we delete a tab, the items are still visible. Has you can see on this screenshot.

![image](https://github.com/elementor/elementor/assets/5083355/ced95506-599c-4b01-9ef8-650d3aea752e)

And once that bug is introduced in the page, it seems that even duplicating an accordion breaks. Could it be an ID issue?
And once the widget is broken, we can't access/modify the tab content.

### Steps to reproduce

1. Create a tab widget
2. Copy the tab widget, but paste it using Ctrl+V
3. Open the Navigator to see the Accordion
4. Delete a tab in the accordion widget.
5. The item is still there in the Navigator.

### Expected behavior

Copy & paste should not break the accordion widget. Once pasted, we should be able the delete tab without issues.

### Isolating the problem

- [X] This bug happens when only the Elementor (and Elementor Pro) plugins are active.
- [X] This bug happens with the Hello Elementor theme active.
- [X] I can reproduce this bug consistently by following the steps I described above.

### Elementor System Info

<details>
<summary>Click to reveal</summary>

````txt

== Server Environment ==
	Operating System: Linux
	Software: LiteSpeed
	MySQL version: MySQL Community Server - (GPL) v5.7.44
	PHP Version: 7.4.33
	PHP Memory Limit: 512M
	PHP Max Input Vars: 1000
	PHP Max Post Size: 64M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.5.2
	WP Multisite: No
	Max Upload Size: 32 MB
	Memory limit: 40M
	Max Memory limit: 512M
	Permalink Structure: /%year%/%monthnum%/%day%/%postname%/
	Language: en_US
	Timezone: America/Toronto
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor Child
	Version: 2.0.0
	Author: Elementor Team
	Child Theme: Yes
	Parent Theme Name: Hello Elementor
	Parent Theme Version: 3.0.1
	Parent Theme Author: Elementor Team

== User ==
	Role: administrator
	WP Profile lang: en-US
	User Agent: Mozilla/5.0 (X11; Linux x86_64; rv:125.0) Gecko/20100101 Firefox/125.0

== Active Plugins ==
	Burst Statistics - Privacy-Friendly Analytics for WordPress
		Version: 1.6.1
		Author: Burst Statistics B.V.

	Classic Editor
		Version: 1.6.3
		Author: WordPress Contributors

	Complianz Privacy Suite (GDPR/CCPA) premium
		Version: 7.0.9
		Author: Really Simple Plugins

	Duplicator
		Version: 1.5.9
		Author: Duplicator

	Elementor
		Version: 3.21.4
		Author: Elementor.com

	Elementor Pro
		Version: 3.21.2
		Author: Elementor.com

	Loginizer
		Version: 1.8.4
		Author: Softaculous

	Phoenix Media Rename
		Version: 3.11.7
		Author: crossi72

	Pods - Custom Content Types and Fields
		Version: 3.2.1
		Author: Pods Framework Team

	Yoast SEO
		Version: 22.6
		Author: Team Yoast


== Elements Usage ==
	
	footer : 2
		container : 2
		heading : 1
		nav-menu : 2
		share-buttons : 1
		theme-site-title : 2
	header : 2
		container : 2
		text-editor : 2
		theme-site-title : 1
	page : 4
		button : 3
		container : 24
		gallery : 1
		heading : 30
		image : 12
		text-editor : 10
		theme-page-title : 1
	popup : 3
		container : 1
		form : 1
		heading : 2
		text-editor : 1
	wp-post : 0
		button : 1
		container : 9
		heading : 10
		image : 5
		text-editor : 1
	wp-page : 8
		button : 36
		container : 196
		divider : 10
		form : 4
		gallery : 3
		heading : 196
		html : 4
		icon-list : 2
		image : 55
		image-box : 4
		loop-grid : 28
		menu-anchor : 2
		nav-menu : 1
		nested-accordion : 11
		nested-tabs : 2
		share-buttons : 1
		shortcode : 1
		spacer : 3
		text-editor : 91
		theme-page-title : 1
		theme-site-title : 1
		video : 1
	single-post : 2
		container : 4
		heading : 2
		loop-grid : 1
		text-editor : 1
		theme-post-featured-image : 1
		theme-post-title : 1
	loop-item : 11
		button : 1
		container : 25
		heading : 29
		nested-accordion : 4
		shortcode : 1
		text-editor : 5
		theme-post-content : 4
		theme-post-featured-image : 1
		theme-post-title : 1
		video : 1
	error-404 : 1
		button : 1
		container : 2
		heading : 3
		text-editor : 1


== Settings ==
	
	cpt_support: post, page
	allow_tracking: yes
	unfiltered_files_upload: 1
	font_display: swap


== Features ==
	Custom Fonts: 0
	Custom Icons: 0

== Integrations ==
	
	recaptcha_v3: Active


== Elementor Experiments ==
	Improved Asset Loading: Active by default
	Improved CSS Loading: Active by default
	Inline Font Icons: Inactive by default
	Additional Custom Breakpoints: Inactive
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Active
	Upgrade Swiper Library: Active by default
	Grid Container: Active
	Nested Elements Performance: Inactive by default
	Optimized Control Loading: Inactive by default
	Default to New Theme Builder: Inactive
	Hello Theme Header & Footer: Inactive
	Elementor Home Screen: Active by default
	Editor Top Bar: Inactive by default
	Build with AI: Active by default
	Landing Pages: Inactive
	Nested Elements: Active
	Lazy Load Background Images: Inactive by default
	Display Conditions: Inactive by default
	Form Submissions: Active by default
	Menu: Inactive by default
	Taxonomy Filter: Inactive by default


== Log ==
	
Log: showing 20 of 412024-04-25 02:39:49 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.20.2',
  'to' => '3.21.2',
)]
2024-04-25 02:40:07 [info] elementor-pro::elementor_pro_updater Started 
2024-04-25 02:40:07 [info] Elementor Pro/Upgrades - _on_each_version Start  
2024-04-25 02:40:07 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2024-04-25 02:40:07 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.19.2',
  'to' => '3.21.1',
)]
2024-04-26 02:25:03 [info] elementor::elementor_updater Started 
2024-04-26 02:25:03 [info] Elementor/Upgrades - _on_each_version Start  
2024-04-26 02:25:04 [info] Elementor/Upgrades - _on_each_version Finished 
2024-04-26 02:25:04 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.21.2',
  'to' => '3.21.3',
)]
2024-04-26 02:25:04 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.21.2',
  'to' => '3.21.3',
)]
2024-04-30 14:20:27 [info] elementor::elementor_updater Started 
2024-04-30 14:20:27 [info] Elementor/Upgrades - _on_each_version Start  
2024-04-30 14:20:27 [info] Elementor/Upgrades - _on_each_version Finished 
2024-04-30 14:20:27 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.21.3',
  'to' => '3.21.4',
)]
2024-04-30 14:20:27 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.21.3',
  'to' => '3.21.4',
)]
2024-04-30 14:20:41 [info] elementor-pro::elementor_pro_updater Started 
2024-04-30 14:20:41 [info] Elementor Pro/Upgrades - _on_each_version Start  
2024-04-30 14:20:41 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2024-04-30 14:20:41 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.21.1',
  'to' => '3.21.2',
)]
2024-04-30 14:20:41 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.21.1',
  'to' => '3.21.2',
)]

JS: showing 2 of 2JS: 2024-04-26 07:04:34 [error X 14][/wp-content/plugins/elementor/assets/js/0e842fae60df2bcb2a30.bundle.min.js:2:7768] t.target.closest(...) is null 
JS: 2024-04-26 07:06:57 [error X 1][/wp-content/plugins/elementor/assets/js/frontend.min.js?ver=3.21.3:2:30429] elementor.settings.page is undefined 



== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatibility not specified

== Elementor Pro - Compatibility Tag ==
````

</details>

### Agreement

- [X] I agree that my issue may be closed without action if it doesn't meet all the requirements.

**Comment by tamelander** [comment]
I too have problems, both with the new Accordion and the new Tabs. My guess it's the nesting that is wonky. Since nesting is in Beta, I have been working on, having to reload the page when the console errors add up and the page refuses to save.

In the Navigator: I have found that when adding a new accordion, the default three tabs are Tab # 1, Tab # 2 and Tab # 3. If I add a tab with the Add tab button, it is named Tab # 4. But if I duplicate an existing tab, e.g. Tab # 2, it is named like its "father": Tab # 2. These Tab titles (Tab # 1 etc) are not editable, I add title strings in the tab's title field but they don't show up in the Navigator, but that is perhaps expected behavior? It doesn't look that good with the tabs showing as e.g.
Tab #1
Tab #2
Tab #3
Tab #2
Tab #2
Tab #2
Tab #2
Tab #2

**Comment by tamelander** [comment]
I always get this error message when loading an Elementor page.
frontend.min.js?ver=3.21.3:2 Uncaught TypeError: e.O is not a function

I added a fresh accordion to a page. Duplicated some tabs. Copied a text widget and pasted into the first item. Three error messages:

**Comment by tamelander** [comment]
I always get this error message when loading an Elementor page.
frontend.min.js?ver=3.21.3:2 Uncaught TypeError: e.O is not a function

I added a fresh accordion to a page. Duplicated some tabs. Copied a text widget and pasted into the first item. Three more error messages: 

TypeError: Cannot read properties of undefined (reading 'id') web-cli.min.js?ver=3.21.3:3

TypeError: Cannot read properties of undefined (reading 'id') web-cli.min.js?ver=3.21.3:3

Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'id') ai.min.js?ver=3.21.3:3

A load of warnings as usual, I don't bother with them anymore though they are irking me a bit.

perfect-scrollbar.min.js?ver=1.4.0:6 [Violation] Added non-passive event listener to a scroll-blocking 'wheel' event. Consider marking event handler as 'passive' to make the page more responsive. See https://www.chromestatus.com/feature/5745543795965952

dev-tools.min.js?ver=3.21.3:2    `container.findChildrenRecursive( callback )` is hard deprecated since 3.5.0 - Use `container.children.findRecursive( callback )` instead

web-cli.min.js?ver=3.21.3:3    `requireContainer()` is hard deprecated since 3.7.0 - Use `Extend `$e.modules.editor.CommandContainerBase` or `$e.modules.editor.CommandContainerInternalBase`` instead

**Comment by tamelander** [comment]
I can provide login credentials to a staging site, if somebody from Elementor wants to "bug hunt".

**Comment by nicholaszein** [comment]
@tamelander,

please don't comment unrelated issues as comments. Follow the guidelines for opening a new [🐞 Bug Report](https://github.com/elementor/elementor/issues/new?assignees=&labels=status%2Fawaiting_triage&template=bug.yml), so we can help you better.

**Comment by nicholaszein** [comment]
Hello @vgstef !

Thank you for your submission! 🙏 

### 🔗 🐞 This issue is a Duplicate of #25861

Which is on the same subject. 

For this reason, I'm closing this thread and 🔃 **Merging** it with the original, so we can have all the discussions in one location, and deal with it more efficiently.

Feel free to continue the conversation there. 😉 

### 🔍 Please, in the future, remember to check if there isn't any thread similar to yours already opened. 

Kind regards

**Comment by nicholaszein** [comment]
This issue was solved in Elementor v3.21.5
